jsonParser.go

```
@@ -0,0 +1,195 @@
+package crossrefindexer
+
+import (
+	"encoding/json"
+	"github.com/davecgh/go-spew/spew"
+	"io"
+	"log"
+	"os"
+	"time"
+)
+
+var i int = 0
+
+func JsonParser() {
```

I don't think you have thought enough about application composition. 
I would suggest that this file should take a `io.Reader` as input and should at least return an error, but probably also a way to extract data from it

---

You are correct, I mainly focused on the behavior of the decoding process.